lucien-ios/Add New Comic Book/AddComicViewController.swift

```diff
@@ -0,0 +1,224 @@
+    @IBOutlet private weak var volumeLabel: UILabel!
+    @IBOutlet private weak var volumeTextField: UITextField!
+    @IBOutlet private weak var issueLabel: UILabel!
+    @IBOutlet private weak var issueTextField: UITextField!
+    @IBOutlet private weak var publisherLabel: UILabel!
+    @IBOutlet private weak var publisherTextField: UITextField!
+    @IBOutlet private weak var releaseDateLabel: UILabel!
+    @IBOutlet private weak var genreLabel: UILabel!
+    @IBOutlet private weak var conditionLabel: UILabel!
+    @IBOutlet private weak var selectADateButton: UIButton!
+    @IBOutlet private weak var selectAGenreButton: UIButton!
+    @IBOutlet private weak var selectAConditionButton: UIButton!
+    @IBOutlet private weak var datePicker: UIDatePicker!
+    @IBOutlet private weak var conditionPicker: UIPickerView!
+    @IBOutlet private weak var genrePicker: UIPickerView!
+    @IBOutlet private weak var scrollView: UIScrollView!
+    @IBOutlet private weak var releaseDateBottomLine: UIView!
+    @IBOutlet private weak var genreBottomLine: UIView!
+    @IBOutlet private weak var conditionBottomLine: UIView!
+    @IBOutlet private weak var comicTitleWarningLabel: UILabel!
+
+    // MARK: - Private Variables
+    private var didCompleteForm = false
+    private var finishButton = UIBarButtonItem()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureNavigationController()
+        configureViewController()
+    }
+
+    // MARK: - Private Instance Methods
+
+    @objc private func backButtonTapped() {
+        dismiss(animated: true, completion: nil)
+    }
+
+    private func configureNavigationController() {
+        guard let navigationController = navigationController else {
+            return
+        }
+
+        // NavigationBar Background
+
+        navigationController.navigationBar.isTranslucent = true
+        navigationController.navigationBar.barStyle = .default
+        navigationController.navigationBar.barTintColor = LucienTheme.white
+        navigationController.navigationBar.setValue(true, forKey: "hidesShadow")
+
+        // NavigationBar Title
+        navigationController.viewControllers[0].title = "Add Book"
+        navigationController.navigationBar.titleTextAttributes = [NSAttributedStringKey.font: LucienTheme.Fonts.PermanentMarker.regular.getFont(size: 30)]
+
+        // NavigationBar Back Button
+        let backButton = UIBarButtonItem(image: UIImage(named: "navBackButton"), style: .plain, target: self, action: #selector(AddComicViewController.backButtonTapped))
+        backButton.tintColor = UIColor.black
+        navigationItem.leftBarButtonItem = backButton
+
+        // NavigationBar 'Finish' Button
+        finishButton = UIBarButtonItem(title: "Finish", style: .plain, target: self, action: nil)
+        finishButton.setTitleTextAttributes([NSAttributedStringKey.font: LucienTheme.Fonts.Muli.semiBold.getFont(size: 17)], for: .normal)
```

Comment:
Having this `getFont` method is fine, but I'd suggest helper methods in theme for the most common fonts so you can just call `LucienTheme.muliSemiBold(size: 17)` which is a little less verbose.